mips/malta.c

```diff
@@ -0,0 +1,64 @@
+#include <mips/malta.h>
+#include <malloc.h>
+
+void clear_bss() {
+  extern unsigned int __bss[];
+  extern unsigned int __ebss[];
+  for (unsigned int *p = __bss, *end = __ebss; p < end; ++p) *p = 0;
```

Comment:
Please **do use** functions from [stdc.h](https://github.com/cahirwpz/mimiker/blob/master/include/stdc.h) (`bzero` in this case). If there're functions from C standard library you'd like to use inside the kernel, please ask about them.